contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+    :to="to"
+  >
+    <div>
+      <KImg
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+        :src="thumbnailSrc"
+        :height="300"
+        :width="500"
+        :isDecorative="true"
+        :style="KImgVerticalSmallStyle"
+      />
+      <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+      <div class="horizontal-layout-style">
+
+        <KImg
+          v-if="layout === 'horizontal' && thumbnailDisplay === 'large'"
+          :src="thumbnailSrc"
+          :height="300"
+          :width="600"
+          :isDecorative="true"
+        />
+        <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+        <div>
+          <div
+            v-if="thumbnailDisplay === 'none'"
+            class="spacing"
+          >
+            <!-- @slot Places content to be placed above the title.. -->
+            <slot name="aboveTitle"></slot>
+            <!-- @slot for the title content -->
+            <slot name="title"></slot>
+            <!-- @slot  Places content below the title.. -->
+            <slot name="belowTitle"></slot>
+          </div>
+          <KGrid
+            v-else
+          >
+            <KGridItem
+              :layout12="{ span: isVerticalLayout ? 12 : 6 }"
+              :layout8="{ span: isVerticalLayout ? 8 : 4 }"
+              :layout4="{ span: 4 }"
+            >
+              <div
+                class="spacing"
+                :style="textColor"
+              >
+                <!-- @slot Places content to be placed above the title.. -->
+                <slot name="aboveTitle"></slot>
+                <!-- @slot for the title content -->
+                <slot name="title"></slot>
+                <!-- @slot  Places content below the title.. -->
+                <slot name="belowTitle"></slot>
+              </div>
+            </KGridItem>
+
+            <KGridItem
+              v-if="layout === 'horizontal'"
+              :layout12="{ span: 6 }"
+              :layout8="{ span: 4 }"
+              :layout4="{ span: 4 }"
+            >
+              <KImg
+                v-if="thumbnailDisplay === 'small'"
+                :src="thumbnailSrc"
+                :height="300"
+                :width="300"
+                :isDecorative="true"
+                :appearanceOverrides="{
+                  scaleType: thumbnailScaleType,
+                }"
+                style="border-radius:20px;"
+              />
+              <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+            </KGridItem>
+          </KGrid>
+
+          <div class="spacing">
+            <slot name="footer"></slot>
+          </div>
+        </div>
+      </div>
+    </div>
+  </BaseCard>
```

Comment:
No issue, but I just wanted to note the significance of semantic elements, for future consideration. Consider this structure (simplified for example):
```html
<section>
  <header>
    <!-- titles and other header slots -->
  </header>
  <aside>
    <!-- thumbnail -->
  </aside>
  <div>
    <!-- main content -->
  </div>
  <footer>
    <!-- footer buttons etc. -->
  </footer>
</section>
```

Reply:
Thanks! Updating it as well.

Reply:
Thank you @bjester. I too thought this would be good so I asked @AllanOXDi to incorporate it, but I think neither of us realized what @radinamatic mentions in this comment https://github.com/learningequality/kolibri-design-system/pull/625#issuecomment-2115805067, so I believe this is being reverted.